app/src/main/java/com/doctor/yumyum/presentation/ui/write/WriteTagActivity.kt

```diff
@@ -75,13 +85,34 @@ class WriteTagActivity : BaseActivity<ActivityWriteTagBinding>(R.layout.activity
         binding.writeTagTvGuide.text = guideText
     }
 
+    fun removeClickListener() {
+        when (removeMode) {
+            DELETE_STATUS -> {
+                removeMode = SELECT_DELETE_STATUS
+                binding.writeTagTvRemove.text = resources.getString(R.string.write_tag_select_delete)
+                binding.writeTagTvRemove.setTextColor(resources.getColor(R.color.main_orange))
+            }
+            SELECT_DELETE_STATUS -> {
+                removeMode = DELETE_STATUS
+                binding.writeTagTvRemove.text = resources.getString(R.string.write_tag_delete)
+                binding.writeTagTvRemove.setTextColor(resources.getColor(R.color.black))
+
+                removeItemList.forEach {
+                    tagViewModel.removeTagItem(it)
+                }
+                removeItemList.clear()
+            }
+        }
+    }
+
     fun finishInput() {
         val intent = Intent(this, WriteFragment2::class.java)
+        intent.putStringArrayListExtra("inputList", tagViewModel.tagListLiveData.value)
         setResult(requestCode, intent)
         finish()
     }
 
-    fun cancleInput(){
+    fun cancleInput() {
```

Comment:
```suggestion
    fun cancelInput() {
```
근데 이거 어디에서 쓰이는 함수인가요?? 굳이 함수로 정의하지 않고 필요한 곳에서 바로 `finish()`로 처리해도 될 것 같아용

Reply:
얘는 xml에서 취소버튼 눌렀을 때 호출하는 함수여서 따로 빼놓았습니당

Reply:
아하 데이터 관련된 부분은 아니니까 xml에서 처리하지 않고 액티비티에서 바로 `binding.button.setOnClickListener { finish() }`로 처리하는 건 어떨까요?

Reply:
넵 그게 좋을거 같아요! 수정하겠습니당